pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -337,3 +325,14 @@ func (f *Fit) Score(ctx context.Context, state *framework.CycleState, pod *v1.Po
 
 	return f.score(pod, nodeInfo)
 }
+
+// InsufficientResource describes what kind of resource limit is hit and caused the pod to not fit the node.
```

Comment:
why did you move this? If unintentional, please revert.

Reply:
fine